grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts

```diff
@@ -23,11 +23,12 @@ export class CustomClrColumnOrderingColumnDirective {
   }
 
   @HostListener('keydown', ['$event']) keydown(event: KeyboardEvent) {
+    const isColumnTarget = (event.target as HTMLElement)?.tagName === 'CLR-DG-COLUMN';
```

Comment:
I think this should be in a separate commit that explains what the change does.

Reply:
fixed